src/Randomizer.SMZ3.Tracking/AutoTracking/AutoTrackerModules/FinalBossCheck.cs

```diff
@@ -50,7 +51,7 @@ private void CheckBeatFinalBosses(SnesData data, SnesData? prevData)
                 {
                     Logger.LogInformation("Auto tracked Mother Brain");
                     Tracker.MarkBossAsDefeated(motherBrain, admittedGuilt: true, confidence: null, autoTracked: true);
-                    CountHyperBeamShots();
+                    _ = CountHyperBeamShots();
```

Comment:
Just as an FYI, I'm pretty sure this will silently swallow exceptions. To be fair though, I don't know of an easy to way to handle fire-and-forget async calls properly.

Reply:
I could probably catch exceptions in the async method itself so they at least get written out to the logs.